### DISCUSSION TITLE
How do I build a vdproj (installer) from a YML file?

### DISCUSSION THREAD

**Discussion by JKochman** [actions/virtual-environments#5844] [General questions] [answered]
I have a Visual Studio 2017 project that has C++, C#, C++/CLI and Installers ...
all the projects load but the installers (vdproj projects ) do not build ...

name: xxx-github-actions
on: [push]
jobs:
  build-release-configuration:
    runs-on: self-hosted
    steps:
      - name: checkout
        uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: '14'
      - name: Add msbuild to Path
        uses: microsoft/setup-msbuild@v1
        with:
          vswhere-path: 'C:\Program Files (x86)\Microsoft Visual Studio\Installer\'
          vs-version: '[15.9,16.0)'        

      - uses: nuget/setup-nuget@v1
        with:
         nuget-version: 4.7.1
      - run: nuget restore xxx.project.sln  
        
      - run: msbuild xxx.project.sln  /p:Configuration=Release /p:Platform=x64

**Comment by al-cheb** [comment, marked as answer]
Please ask general questions to https://github.community/ forum.

**Reply by JKochman** [reply]
done
